internal/locate/cse/cse.go

```diff
@@ -0,0 +1,428 @@
+package cse
+
+import (
+	"bytes"
+	"context"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"sync"
+	"time"
+
+	"github.com/golang/protobuf/proto" //nolint:staticcheck
+	"github.com/pingcap/kvproto/pkg/metapb"
+	"github.com/pingcap/kvproto/pkg/pdpb"
+	"github.com/pingcap/log"
+	"github.com/pkg/errors"
+	"github.com/tikv/client-go/v2/metrics"
+	"github.com/tikv/client-go/v2/tikvrpc"
+	"github.com/tikv/client-go/v2/util"
+	"github.com/tikv/client-go/v2/util/codec"
+	pd "github.com/tikv/pd/client"
+	pdcore "github.com/tikv/pd/pkg/core"
+	"go.uber.org/zap"
+)
+
+var (
+	_ pd.Client = &Client{}
+)
+
+var (
+	StoresRefreshInterval = time.Second * 5
+	SyncRegionTimeout     = time.Second * 2
```

Comment:
how about `time.Second * 10`?

Reply:
This duration might be too long for the newly created cluster.